code/controllers/subsystem/SSeconomy.dm

```diff
@@ -92,6 +92,8 @@ SUBSYSTEM_DEF(economy)
 	var/next_paycheck_delay = 0
 	/// total paydays this round
 	var/payday_count = 0
+	/// Time until the next mail shipment
+	var/next_mail_delay = 0
```

Comment:
Could use TG cooldown, but only if you feel like doing it.

Reply:
Really not in the mood for this honestly, could be taken later by a larger PR that replaces most of our timers with TG cooldowns

Reply:
Yep, this can be handled later. The code works as-is for now.